docs/cli-usage.md

```diff
@@ -47,7 +47,7 @@ djinni \
 | `--java-class-access-modifier <public/package>` | The access modifier to use for generated Java classes (default: `public`). |
 | `--java-cpp-exception <exception-class>` | The type for translated C++ exceptions in Java (default: `java.lang.RuntimeException` that is not checked) |
 | `--java-annotation <annotation-class>` | Java annotation (`@Foo`) to place on all generated Java classes |
-| `--java-generate-interfaces <true/false>` | Whether Java interfaces should be used instead of abstract classes where possible (default: false). |
+| `--java-generate-interfaces <true/false>` | Whether Java interfaces should be used instead of abstract classes where possible (default: `false`). |
```

Comment:
There is a few of these changes in this PR. These are just fixes for formatting errors. Default values should always be put in backticks.